### PR TITLE
Add webpage for user submission of URL

### PR DESCRIPTION
**Pull request by duwerq**
Add html views for webpage submission.  Still working on passing url submission to parse_webpage method.

### CODE REVIEW

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -48,11 +53,12 @@ def destroy
     head :no_content
   end
 
-  def parse_webpage
+  def parse_webpage(website)
     require 'open-uri'
```
Comment (ike4est) [inline app/controllers/tasks_controller.rb]: these require statements should go at the top of the controller

File: app/controllers/tasks_controller.rb [hunk 2]
```diff
@@ -13,19 +14,23 @@ def index
...
   # POST /tasks
   # POST /tasks.json
   def create
-    @task = Task.new(task_params)
 
-    if @task.save
-      render json: @task, status: :created, location: @task
-    else
-      render json: @task.errors, status: :unprocessable_entity
+    if params[:task][:website]
```
Comment (ike4est) [inline app/controllers/tasks_controller.rb]: task_params[:website]

File: app/controllers/tasks_controller.rb [hunk 3]
```diff
@@ -13,19 +14,23 @@ def index
...
+      parse_webpage(params[:task][:website])
```
Comment (ike4est) [inline app/controllers/tasks_controller.rb]: i believe this should be self.parse_webpage i could be wrong though and you might want move this to the modal

**Review by ike44est — CHANGES_REQUESTED** [review]
overall this looks really good!

**Pull request merged**